From: Kyle McMartin <kyle@phobos.i.jkkm.org>
Date: Tue, 30 Mar 2010 00:04:29 -0400
Subject: [PATCH] die-floppy-die

Kill the floppy.ko pnp modalias. We were surviving just fine without
autoloading floppy drivers, tyvm.

Please feel free to register all complaints in the wastepaper bin.

Bugzilla: N/A
Upstream-status: Fedora mustard
---
 drivers/block/floppy.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/block/floppy.c b/drivers/block/floppy.c
index a08cda955285..e320e1e679cf 100644
--- a/drivers/block/floppy.c
+++ b/drivers/block/floppy.c
@@ -5014,8 +5014,7 @@ static const struct pnp_device_id floppy_pnpids[] = {
 	{ .id = "PNP0700" },
 	{ }
 };
-
-MODULE_DEVICE_TABLE(pnp, floppy_pnpids);
+/* MODULE_DEVICE_TABLE(pnp, floppy_pnpids); */
 
 #else
 
